### PR TITLE
doc:  minutes for deep dives at Jun Collab summit

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
Still need to get the link for the recording. Looked in Node.js youtube but did not find it.

Reached out to @waleed to see if he knows were it is.

File: meetings/summit-jun-2022.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+* Geoffrey - Who is using ESM in a real production app where ESM is the entry point
+  * Only James/Geoffrey
+  * Datadog only has 2 out of 10ths of thousands use it, import in the middle satisfied those 2.
+  * Anybody using instrumentation with it?
+  * Vercel supports ESM but converts to CJS
+  * Those who write ESM are larger than CJS.
+  * LJHarb: no added value to run ESM in production versus CJS over authoring in ESM which is already possible/being done, even when deployment is then CJS.
+  * Cloudflare is switching over to ESM for entrypoint as another datapoint
+  * Matteo, one use case that was good for ESM, lamda with top level await, with reserved capacity, can move pre-loading
```
Comment (sheplu) [inline meetings/summit-jun-2022.md:128]:
```suggestion
  * Matteo, one use case that was good for ESM, lambda with top level await, with reserved capacity, can move pre-loading
```

File: meetings/summit-jun-2022.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+### Observability
+
+* Stephen - some needs from an APM perspective
+  * Need better way to be able to run code before application starts
+    * -r /loaders
+    * Geoffrey, –import command line as well
+    * Ordering with loaders is an issue (loaders already work with both ESM and CommonJS)
+    * Issues with serverless, need co-operation with serverless vendors
+    * Node clinic, need to load before before event loop starts, if start later then miss some objects
```
Comment (sheplu) [inline meetings/summit-jun-2022.md:163]:
```suggestion
    * Node clinic, need to load before event loop starts, if start later then miss some objects
```

**Review by sheplu — CHANGES_REQUESTED** [review]
Just a few typo / changes then I think it will be good to be merged

File: meetings/summit-jun-2022.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+* Long stack trace support
+  * Asking V8 for long stack trace support that is not connected to async await (for libuv continuations).
+    Stephen has been asking for a clearly defined task object and track their stuff and our stuff,
+    and passed back on async resume.
+
+#### Summary of what we agreed
+
+* Priorities for Observability
+  * Diagnostic Channel (common with ESM)
+  * Better advocacy to module owners, primining them for PRs from APMs etc and communicating best pratices
```
Comment (trivikr) [inline meetings/summit-jun-2022.md:214]: priming?
```suggestion
  * Better advocacy to module owners, priming them for PRs from APMs etc and communicating best pratices
```

File: meetings/summit-jun-2022.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+    * Create doc that lists where we think data should be sent to diagnostic channel so that others
+      can help add generation of data at those points.
+    * Trevor is going to PR in additional to Diagnostic Channel for a number of places NodeSource is already
+      capturing data.
+  * Continue work on loaders (common with ESM)
+    * Jacob is progressing and moving forward, will get completed but could be completed a
+      lot sooner if there was more help
+    * Should we make this a strategic initiative to increase visibility/support at TSC level?
+    * Goal is to get loaders to non-experimental this year
+  * Initiate work on enhanded startup lifecycle than can be used to ensure Observability tools are loaded
```
Comment (trivikr) [inline meetings/summit-jun-2022.md:231]:
```suggestion
  * Initiate work on enhanced startup lifecycle than can be used to ensure Observability tools are loaded
```

**Review by sheplu — APPROVED** [review]

**Review by trivikr — APPROVED** [review]

**Pull request merged**